code-challenges.test.js

```diff
@@ -28,29 +64,81 @@
+                // Using typeof, only return the value that is not a string, boolean, null. And only return the value that is odd. 
+                // Return the array sorted from least to greatest using the buit in method .sort().
+const findOdds = (array) => {
+let oddOnly =  array.filter(value => {
+        return typeof value !== "string" && typeof value !== "boolean" && typeof value !== null && value % 2 !== 0 
+    })
+    return oddOnly.sort((a,b) => a - b)
+    
+}
+
 
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const numbersToAdd1 = [2, 4, 45, 9]
-// Excpected output: [2, 6, 51, 60]
-
-const numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
-
-const numbersToAdd3 = []
-// Expected output: []
+//Psuedo-Code for Jest Test:
+        // A describe method that lists the name of the function OR naming of the particular test.
+        describe("accSum", () => {
+            // A test/it method, nested within the describe block, that in plain words, describe what the function does. 
+            it("takes in an array and returns an array of the accumulating sum. If it is an empty array, it should return an empty array back.", () => {
+                const numbersToAdd1 = [2, 4, 45, 9]
+                // Excpected output: [2, 6, 51, 60]
+                
+                const numbersToAdd2 = [0, 7, -8, 12]
+                // Expected output: [0, 7, -1, 11]
+                
+                const numbersToAdd3 = []
+                // Expected output: []
+            
+               // An expect method, nested within the test block, calling on findOdds function, followed by the toEqual() matcher that checks the expected output of the function return. Create 2 expects because there are two arrays to be passed as arguments. 
+               expect(accSum(numbersToAdd1)).toEqual( [2, 6, 51, 60] )
+               expect(accSum(numbersToAdd2)).toEqual( [0, 7, -1, 11] )
+               expect(accSum(numbersToAdd3)).toEqual( [] )
+            })
+         })
 
 
 // b) Create the function that makes the test pass.
+         //Psuedo-Code: 
+            // Create a function called accSum that takes in an array as a parameter. 
+            // Start by declaring a variable num equal to zero because at first instance no number is accumlating.
+            // Using the built in method .map(),it will iterate through each value in the array. 
+            // Reassign num each time so that value + num is the next value in the array. It is constantly reassigning num to new value. 
+            // Return sum1 
+const accSum = (array) => {
+   let num = 0
+    let sum1 = array.map(value => {
+     num = value + num
+    return num
+    })
+    return sum1
+}
```

Comment:
Excellent!  You can simplify this down even further by replacing your variable declaration for a return on line 139 and removing 143.